database/migrations/2016_09_26_093633_document.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class Document extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('models', function (Blueprint $table) {
```

Comment:
This isn't really the best way to handle this, since the very next thing people will ask for is multiple uploads, which means this schema won't work (or we'll need to write a migration to move the data to a model_uploads table or something else.) I guess this is okay for now, but I know we're going to have to change it.